src/main/scala/EnsimePlugin.scala

```diff
@@ -370,34 +384,31 @@ object EnsimePlugin extends AutoPlugin {
     }
 
     if (scalaVersion.gimme != ensimeScalaV) {
-      if (System.getProperty("ensime.sbt.debug") != null) {
-        // for testing
-        IO.write(file("scalaVersionAtStartupWarning"), ensimeScalaV)
-      }
-
-      log.error(
+      log.warn(
         s"""You have a different version of scala for ENSIME ($ensimeScalaV) and ${project.id} (${scalaVersion.gimme}).
-           |If this is not what you intended, use either
+           |If this is not what you intended, try fixing your build with:
            |  scalaVersion in ThisBuild := "${scalaVersion.gimme}"
-           |in your build.sbt or add
+           |in your build.sbt, or override the ENSIME scala version with:
            |  ensimeScalaVersion in ThisBuild := "${scalaVersion.gimme}"
-           |to a local ensime.sbt: http://ensime.org/build_tools/sbt/#customise""".stripMargin
+           |in a ensime.sbt: http://ensime.org/build_tools/sbt/#customise""".stripMargin
       )
+      if (!ensimeIgnoreScalaMismatch.gimme)
+        throw new IllegalStateException(
+          """To ignore this error (i.e. have have multiple scala versions), customise `ensimeIgnoreScalaMismatch`"""
```

Comment:
argh. any chance you could just make these typo fixups and send a quick PR? they sound good and you can do it directly in github